examples/plotting/plot_rectangle.py

```diff
@@ -0,0 +1,100 @@
+"""
+==========================================================
+Drawing a Rectangle on a map
+==========================================================
+
+In this example we are going to look at how we can plot rectangles on maps
+while describing the dimensions using different methods.
+"""
+
+################################################################################
+# Fundamentally, we will be using the `~sunpy.map.GenericMap.draw_rectangle` method to
+# draw the rectangles. Its function signature is like so:
+#
+# `draw_rectangle(bottom_left, *, width: Unit(‘deg’) = None,
+# height: Unit(‘deg’) = None, axes=None, top_right=None, **kwargs)`
```

Comment:
Remove this. We already link to `draw_rectangle` in the API docs above. There's no need to reproduce that here.